pkgs/by-name/fo/fosrl-olm/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+# versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "olm";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "fosrl";
+    repo = "olm";
+    tag = version;
+    hash = "sha256-a0gkEo5EuJpHpZ5fKAPBXSTRvZaQo6KOJu4Abi1EztU=";
+  };
+
+  vendorHash = "sha256-Ms8tLFKIa8GqmyFI7o+sQEpsZghNPIpq8BRCoY89Org=";
+
+  # todo versioning when upstream adds it
+  # nativeInstallCheckInputs = [ versionCheckHook ];
+  # versionCheckProgramArg = [ "-version" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  doInstallCheck = true;
+
+  meta = {
+    description = "Tunneling client for Pangolin";
```

```suggestion
    description = "Tunneling client for Newt sites";
```
That's Newt, not Olm.